.github/actions/veracode/source_clear.sh

```diff
@@ -4,10 +4,7 @@ echo "=========================== Starting SourceClear Script ==================
 PS4="\[\e[35m\]+ \[\e[m\]"
 set +e -v -x
 
-mvn -B -q clean install \
-    ${SRCCLR_INSTALL_DEFAULT_OPTIONS} ${SRCCLR_INSTALL_OPTIONS} \
-    com.srcclr:srcclr-maven-plugin:scan \
-    -Dcom.srcclr.apiToken=${SRCCLR_API_TOKEN} > scan.log
+srcclr scan --loud > scan.log
```

Comment:
Did we test this on a project that has actual SCA findings, to verify that the `grep ...` instruction few lines before still works as intended and fetches the Veracode Agent Based Scan URL?

Reply:
It does, but the "-e 'Failed'" gets stuff from maven build because srcclr tries a cheeky way of getting a build in a low-cost manner first, before it does a proper one and the first attempt fails.

I've went back on my decision of adding "--loud" there and I followed Giovanni's recommendation to just remove the script and just call the scan in actions step itself.

Truth be told the only thing that this --loud provides over normal call, are maven logs, and unless that fails I don't think we should bother too much. It can always be tested locally if need arises.

Reply:
Also, we get a nicely formatted report without even having to use grep.

Reply:
I'd like to see 1 example of such logs to make sure we get at least the same level of information we had before (the link to hit the Veracode Web UI) 👍

Reply:
Actually, given that the logs contain sensitive information (e.g.: the vulnerabilities we're affected by) and I'm pretty confident any registered user with read access to a GitHub Repository can also read the logs of a GH Action Run, we should retain the logic to redirect the output somewhere else and only grep for the "Full Report Details" in case of failure! cc: @gionn